docs/everloop.md

```diff
@@ -49,87 +53,62 @@ This driver doesn't send any data to a subscribed program.
 
 ### JavaScript example
 
-Enhanced description of the [sample source code](../src/js_test/test_everloop.js).
+This section provides an enhanced description of the relevant parts of the[sample source code](../src/js_test/test_everloop.js).
+Some variables used below are defined in the example (for instance matrixMalosBuilder).
 
+In order to set the LEDs of the Creator you need to perform the following steps.
 
-First, define the address of the MATRIX Creator. In this case we make it be `127.0.0.1`
-because we are connecting from the local host but it needs to be different if we
-connect from another computer. There is also the base port reserved by MALOS for
-the Everloop driver.
+#### Create a configuration that will be sent to the Everloop driver.
```

Comment:
@arhuaco why not just `Initialize configuration`, or `Initialize config object` as a header?